<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryItem</class>
 <widget class="QWidget" name="queryItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_key"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_equal"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_close">
     <property name="text">
      <string/>
     </property>
     <property name="isIconNoPadding" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
